grpc-gcp/src/main/java/com/google/cloud/grpc/GcpMultiEndpointChannel.java

```diff
@@ -213,6 +215,27 @@ private GcpManagedChannelOptions prepareGcpManagedChannelConfig(
         .build();
   }
 
+  private ManagedChannelBuilder<?> channelBuilderForEndpoint(String endpoint) {
+    String serviceAddress;
+    // Assume https by default.
```

Comment:
What is the schema is not http2?

Reply:
The schema here is used just to determine the port if it is not specified. We will not pass the schema further.